app/assets/javascripts/application.js

```diff
@@ -12,6 +12,8 @@
 //
 //= require angular/angular
 //= require angular-route/angular-route
+//= require ng-token-auth/dist/ng-token-auth
```

Comment:
You remove the authentication from the backend but you still left it in the frontend?